mobile/src/main/java/org/openhab/habdroid/util/CacheManager.kt

```diff
@@ -38,19 +37,37 @@ class CacheManager private constructor(appContext: Context) {
         // int in its constructor.
         val maxMemory = (Runtime.getRuntime().maxMemory() / 1024).toInt()
         // Use up to 1/8 of the available VM memory for the bitmap cache
-        bitmapCache = object : LruCache<HttpUrl, Bitmap>(maxMemory / 8) {
+        iconBitmapCache = object : LruCache<HttpUrl, Bitmap>(maxMemory / 8) {
```

Comment:
Nit: Maybe make this a named class (static class inside CacheManager) instead of duplicating the anonymous class?